Python/pywarpx/picmi.py

```diff
@@ -1126,7 +1119,11 @@ def initialize_inputs(self):
 class FieldIonization(picmistandard.PICMI_FieldIonization):
     def initialize_inputs(self):
         assert self.model == 'ADK', 'WarpX only has ADK ionization model implemented'
```

Comment:
Can you add the WarpX specific doc string for this class that states that only ADK is supported here?

Reply:
I've just added a docstring which I hope is what is needed here.